### PR TITLE
Fixed #47.

### LINKED ISSUE #47: euid in confirm.php is not set
Hi,

if clicking a "confirm all" link in one of the status mails, which lead in my case to

`<domainwashere>/mail/confirm.php?id=70&euid=70&ts=2016-06-09%2000%3A00%3A01&token=<tokenwashere>&report_ham=true`

we get an error like 
`[Mon Jun 13 22:40:26.670741 2016] [:error] [pid 5603] [client <internal>] PHP Fatal error:  Call to undefined method MDB2_Error::fetchRow() in /var/www/html/confirm.php on line 131`
and a white page for the user

tracking it down, an unset $euid was passed to the query above the line.

caused by
```
        $select = "SELECT maia_mail.id " .
                  "FROM maia_mail, maia_mail_recipients " .
                  "WHERE maia_mail.id = maia_mail_recipients.mail_id " .
                  "AND maia_mail.received_date <= ? " .
                  "AND maia_mail_recipients.type = 'H' " .
                  "AND maia_mail_recipients.recipient_id = ?";
        $sth = $dbh->query($select, array($cutoff_date, $euid));
```

the query can't complete, so an unhandled error is thrown which results in a blank page for the user, not reporting mails as ham and the logline above.

### PR DESCRIPTION
**Pull request by piwikjapan**

### CODE REVIEW

**Comment by piwikjapan** [comment]
Fixed incorrect "Maia Quarantine Digest". Delete codes for debug.
For a long time, send-quarantine-digests.pl banned_file treated as virus, banned_file and bad_header.

**Comment by hamstercnw** [comment]
Hello
Check line 234 in send-quarantine-digest.pl.
I believe variable "$elemengt" is a typo.

File: scripts/send-quarantine-digests.pl [hunk 1]
```diff
@@ -231,7 +231,7 @@
 
 my %report_statements;
 while (my($element, $sort) = each(%sort)) {
-    next if exists $report_statements{$sort};
+    next if exists $report_statements{$elemengt};
```
Comment (hamstercnw) [inline scripts/send-quarantine-digests.pl]: Is "$elemengt" a typo?

**Comment by piwikjapan** [comment]
Sorry. Fixed typo.

**Pull request merged**